nautobot/docs/installation/migrating-from-netbox.md

```diff
@@ -55,3 +55,18 @@ Depending on the complexity of your scripts and reports, and how tightly integra
 - `tenancy.* -> nautobot.tenancy.*`
 - `utilities.* -> nautobot.utilities.*`
 - `virtualization.* -> nautobot.virtualization.*`
+
+## Data Model Changes
+
+The following backwards-incompatible changes have been made to the data model.
```

Comment:
I am thinking it would be good to have them tracked here centrally, as the release notes start to fade into history.

We might need to decide for how long we intend to maintain this guide. As I see it, it's a "for now", and not a "forever".

Reply:
You know what. I was going to do that but decided to leave it for another PR.